es6-shim.js

```diff
@@ -719,6 +719,7 @@
   };
   var nonWS = ['\u0085', '\u200b', '\ufffe'].join('');
   var nonWSregex = new RegExp('[' + nonWS + ']', 'g');
+  var isBadHexRegex = new RegExp('^[-+]0x[0-9a-f]+$', 'i');
```

Comment:
Understood, but I seem to remember some older environments had problems with certain literals, I think it was `XRegExp` that pointed these out. :)